x-pack/plugins/lens/public/visualizations/partition/to_expression.ts

```diff
@@ -286,7 +290,7 @@ function expressionHelper(
     }))
     .filter((o): o is { columnId: string; operation: Operation } => !!o.operation);
 
-  if (!layer.metric || !operations.length) {
```

Comment:
out of curiosity - we were allowed to remove check on operations?

Reply:
This was a check for a non-zero count of bucket dimensions. You can now have zero bucket dimensions when you have multiple metrics, so I removed it.

However, I'm glad you asked because I'm realizing now that this leads to a slight change in behavior when the user hasn't enabled multiple metrics and adds a single metric with no bucket dimensions. Before, no visualization would have been rendered. Now, they get a pie chart with no slices.

So, I'll change this check to 
```ts
  if (!layer.metrics.length || (!operations.length && !layer.allowMultipleMetrics)) {
    return null;
  }
```